articles/search/search-region-support.md

```diff
@@ -35,105 +35,104 @@ You can create an Azure AI Search resource in any of the following Azure public
+| Region | AI integration | Semantic ranker | Availability zones | Capacity constrained |
+|--|--|--|--|--|
+| Brazil South​​ ​ | ✅ | ✅ | |  |
+| Canada Central​​ | ✅ | ✅ | ✅ |  |
+| Canada East​​ ​ |  | ✅ | |  |
+| East US​ <sup>1</sup>| ✅ | ✅ | ✅ |  |
+| East US 2 ​ | ✅ | ✅ | ✅ |  <span class='red-x'>&#10060;</span>|
+| ​Central US​​ <sup>1</sup> | ✅ | ✅ | ✅ |  |
+| North Central US​ ​ | ✅ | ✅ | |  | 
+| South Central US​ <sup>1</sup>​ | ✅ | ✅ | ✅ | <span class='red-x'>&#10060;</span> |
+| West US​ ​ | ✅ | ✅ | |  |
+| West US 2​ ​ | ✅ | ✅ | ✅ | |
+| West US 3​ <sup>1</sup>​ | ✅ | ✅ |✅ |  <span class='red-x'>&#10060;</span> |
+| West Central US​ ​ | ✅ | ✅ | | |
+
+<sup>1</sup> Capacity constraints apply to specific tiers. [See tier availability by region for details] (search-sku-tier.md#region-availability-by-tier).
 
 ### Europe
 
-| Region | AI integration | Semantic ranker | Availability zones |
-|--|--|--|--|
-| North Europe​​ | ✅ | ✅ | ✅ |
-| West Europe​​ <sup>1, 2</sup>| ✅ | ✅ | ✅ |
-| France Central​​ | ✅ | ✅ | ✅ |
-| Germany West Central​ ​| ✅ |  | ✅ |
-| Italy North​​ |  |  | ✅ |
-| Norway East​​ | ✅ |  | ✅ |
-| Poland Central​​ |  |  |  |
-| Spain Central <sup>2</sup> |  |  | ✅  |
-| Sweden Central​​ | ✅ |  | ✅ |
-| Switzerland North​ | ✅ | ✅ | ✅ |
-| Switzerland West​ | ✅ | ✅ | ✅ |
-| UK South​ | ✅ | ✅ | ✅ |
-| UK West​ ​|  | ✅ | |
-
-<sup>1</sup> Currently, this region is at full capacity and not accepting new search services.
+| Region | AI integration | Semantic ranker | Availability zones | Capacity constrained |
+|--|--|--|--|--|
+| North Europe​​ | ✅ | ✅ | ✅ |  |
+| West Europe​​ <sup>1, 2</sup>| ✅ | ✅ | ✅ | <span class='red-x'>&#10060;</span> |
+| France Central​​ | ✅ | ✅ | ✅ | |
+| Germany West Central​ ​| ✅ |  | ✅ | |
+| Italy North​​ |  |  | ✅ | |
+| Norway East​​ | ✅ |  | ✅ |  |
+| Poland Central​​ |  |  |  |  |
+| Spain Central <sup>2</sup> |  |  | ✅  |  |
+| Sweden Central​​ | ✅ |  | ✅ |  |
+| Switzerland North​ | ✅ | ✅ | ✅ |  |
+| Switzerland West​ | ✅ | ✅ | ✅ |  |
+| UK South​ | ✅ | ✅ | ✅ |  |
+| UK West​ ​|  | ✅ | |  |
+
+<sup>1</sup> Capacity constraints apply to specific tiers. [See tier availability by region for details] (search-sku-tier.md#region-availability-by-tier).
 
 <sup>2</sup> This region runs on older infrastructure that has lower storage limits per partition at every tier. Choose a different region if you want [higher limits](search-limits-quotas-capacity.md#service-limits).
 
 ### Middle East
 
-| Region | AI integration | Semantic ranker | Availability zones |
-|--|--|--|--|
-| Israel Central​ <sup>2</sup> |  |  | ✅  |
-| Qatar Central​ <sup>1, 2</sup> |  |  | ✅ |
-| UAE North​​ | ✅ |  | ✅ |
+| Region | AI integration | Semantic ranker | Availability zones | Capacity constrained |
+|--|--|--|--|--|
+| Israel Central​ <sup>2</sup> |  |  | ✅  |  |
+| Qatar Central​ <sup>1, 2</sup> |  |  | ✅ | |
```

Comment:
No red x for Qatar?

Reply:
we just enabled that one in the portal yesterday